### PR TITLE
BaSP-TG-99: Superadmin flow

### PR DESCRIPTION
**Pull request by Javier-Cervera**
## Brief description

User Superadmin screens.

We should be able to log into the app as a SuperAdmin. When logged in, we should be able to log out, blocking access to that user by URL.

Also, we should be able to create, edit, and delete (logical delete) an Admin.

## Considerations

For logical deletion we need TG-100 merged, but for practical purposes is the same. As it is now, we can delete an Admin from the table and the DB

### CODE REVIEW

File: src/Components/SuperAdmin/Admins/index.jsx [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+      {modalEdit}
+      {modalAdd}
+      {modalDelete}
+      {modalErrorSuccess}
+      {isLoading ? <Preloader /> : null}
+      <div className={styles.divContainer}>
+        <ButtonText label="ADD ADMIN" clickAction={openAddModal}></ButtonText>
+        <Table
+          data={admins}
+          headers={['_id', 'firstName', 'lastName', 'email', 'active']}
```
Comment (iaracriscenti) [inline src/Components/SuperAdmin/Admins/index.jsx:134]: Is it useful to show the id column?

**Review by iaracriscenti — APPROVED** [review]
Everything works great! 🚀

**Comment by LeonelListro** [comment]
# Test results

Tested locally on: https://alexis-trackgenix-app-git-feature-tg-99-super-25cd2c-basp-m2022.vercel.app/home

test result: failed ⚠️

bugs found:

1- The validation error message in the email input should be displayed.

WhatsApp Image 2022-07-08 at 11 28 23 AM

2- The box of the links in the sidebar should be smaller to facilitate their use. (The sector marked with a red oval is still part of admins link. The same happend with the log out link).

Sin título

**Review by LeonelListro — CHANGES_REQUESTED** [review]
Tested failed. Please refer to https://github.com/BaSP-m2022/alexis-trackgenix-app/pull/55#issuecomment-1179069294

**Review by LeonelListro — APPROVED** [review]
Well done Javier and team! (It is noted that one of the bugs will be addressed in Ticket 108).

**Pull request merged**